src/gui/ar/positioning.js

```diff
@@ -137,28 +199,42 @@ realityEditor.gui.ar.positioning.scaleVehicle = function(activeVehicle, centerTo
  * @param {boolean} useTouchOffset - if false, puts (0,0) coordinate of frame/node at the resulting point.
  *                                   if true, the first time you call it, it determines the x,y offset to drag the frame/node
  *                                   from the ray cast without it jumping, and subsequently drags it from that point
+ * @param {boolean} isContinuous - true to optimize calculations for this being triggered multiple times in a row
+ *                                 if true, you are expected to manually call stopRepositioning when done
  */
-realityEditor.gui.ar.positioning.moveVehicleToScreenCoordinate = function(activeVehicle, screenX, screenY, useTouchOffset) {
+realityEditor.gui.ar.positioning.moveVehicleToScreenCoordinate = function(activeVehicle, screenX, screenY, useTouchOffset, isContinuous) {
     
-    var results;
+    var newPosition;
     try {
-        results = realityEditor.gui.ar.utilities.screenCoordinatesToMatrixXY(activeVehicle, screenX, screenY, true);
+        // set dummy div transform to iframe without x,y,scale
+        if (this.useWebkitForProjectedCoordinaates) {
```

Comment:
typo

Reply:
good catch. my MacBook keyboard is a bit messed up and double-types letters with alarming frequency.